dracula-theme.el

```diff
@@ -344,7 +344,7 @@
    `(which-func ((,class (:inherit ,font-lock-function-name-face))))
    `(dired-directory ((,class (:foreground ,func :weight normal))))
    `(dired-flagged ((,class (:foreground ,keyword))))
-   `(dired-header ((,class (:foreground ,fg3 :background ,func))))
+   `(dired-header ((,class (:foreground ,bg1 :background ,func))))
```

Comment:
I'm not the biggest fan of using a `background` variable for a foreground color, But I didnt want to redeclare the variable. We could also do something like this...

![__code_personal_emacs-1_dracula-theme_el](https://cloud.githubusercontent.com/assets/11561578/22833321/0faa6f1e-ef77-11e6-864a-997f9f29e0d9.png)

Which would be...
```elisp
`(dired-header ((,class (:foreground ,fg3 :background ,bg1))))
```

Thoughts?